README.md

```diff
@@ -1,111 +1,120 @@
+
+This is the Crossplane Provider package for [Packet](https://www.packet.com)
+infrastructure. The provider that is built from the source code in this
+repository can be installed into a Crossplane control plane.
 
 ## Getting Started and Documentation
 
 For getting started guides, installation, deployment, and administration, see [Documentation](https://crossplane.io/docs/latest).
 
-## Pre-requisites 
+## Pre-requisites
 
 * Kubernetes cluster
-    * For example Minikube, minimum version v0.28+
+  * For example Minikube, minimum version v0.28+
 * Helm, minimum version v2.9.1+.
 
 ## Installing Crossplane
 
-Add Helm chart repo:
-```
+For the most up to date, detailed, instructions, check [Crossplane's documentation](https://crossplane.io/docs/v0.12/getting-started/install-configure.html).
+
+The following instructions are provided for convenience.
+
+```console
+kubectl create namespace crossplane-system
 helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install crossplane --namespace crossplane-system crossplane-alpha/crossplane
 ```
-Install Crossplane Helm chart
 
-```
-helm install --name crossplane --namespace crossplane-system crossplane-alpha/crossplane
+### Install the Crossplane CLI
+
+```console
+curl -sL https://raw.githubusercontent.com/crossplane/crossplane-cli/master/bootstrap.sh | bash
 ```
 
-For more details on Crossplane installation visit [here](https://crossplane.io/docs/v0.1/install-crossplane.html)
+## Install the Packet Provider
 
-## Installing `stack-packet` 
+```console
+kubectl crossplane package install --cluster \
+  --namespace crossplane-system \
+  packethost/provider-packet:v0.0.2 provider-packet
+```
 
-* Create CRDs for the stack:
+### Create a Provider Secret
 
-```
-kubectl create -f config/crd
-```
+Create a [Packet Project and a project level API key](https://www.packet.com/developers/docs/API/getting-started/).
 
-```
-customresourcedefinition.apiextensions.k8s.io/providers.packet.crossplane.io created
-customresourcedefinition.apiextensions.k8s.io/deviceclasses.server.packet.crossplane.io created
-customresourcedefinition.apiextensions.k8s.io/devices.server.packet.crossplane.io created
+Run the following and supply the API Key into the console when prompted:
+
+```console
+read -s -p "API Key: " APIKEY; echo
 ```
 
-* Set your Packet authentication token
+Create a Kubernetes secret with this API Key.
 
-From the project root:
-```
-cd cluster/examples
-kubectl create -f namespace.yaml
-./provider.sh
-echo $PACKET_AUTH_TOKEN > credetials.txt #if $PACKET_AUTH_TOKEN is already set
+```console
+kubectl create -n crossplane-system secret generic packet-creds --from-literal=key=$APIKEY
```

Comment:
This implies that the key this provider is looking for is the only contents (string not structure) of the config. Is this the case? Should this be the case (see ProjectID and facility thoughts below).

Reply:
I'm considering replacing this single token string with a string representing the same json body that the [Packet CSI](https://github.com/packethost/csi-packet/blob/e29bd36dbeb52470ce60d68076ee77b5d4f3518b/pkg/packet/provider.go#L23-L30
) and [Packet CCM](https://github.com/packethost/packet-ccm/blob/master/packet/cloud.go#L62-L68) accept:

```json
{
  "projectID": "uuid",
  "apiKey": "uuid",
  "facility-id": "uuid",
  "base-url": "url",
  "metadata-url": "url",
}
```

_json.Unmarshal is case-insensitive when matching tags. The CSI currently mixes hyphenated and mixed-case field tags. FacilityID has a different name between tools.._